draft-ietf-teas-5g-ns-ip-mpls.md

```diff
@@ -1773,7 +1773,7 @@ Also, underlay transports may be realized using separate NRPs. However, such an
    time, unless the 5G NSO communicates a change in slice bandwidth
    requirements, due to the creation or modification of a slice.  Note
    that the "reservations" may be maintained by the transport
-   controller - it is not necessary (or indeed possible for SR-TE) to
+   controller - it is not necessary (or indeed possible for current SR-TE technology) to
```

Comment:
```suggestion
   controller - it is not necessary (or indeed possible for current SR-TE technology per 2024) to
```

Reply:
"current" may not age well in an RFC.

Reply:
Looks good, except I would say "in 2024" rather than "per 2024"